app/views/gov_one/info.html.slim

```diff
@@ -1,14 +1,27 @@
-= render 'learning/cms_debug'
-
 - content_for :page_title do
-  = html_title t('gov_one_info.title')
+  = html_title t('home.title')
+
+- content_for :hero do
+  .govuk-grid-row class='govuk-!-padding-top-9 govuk-!-padding-bottom-0'
+    .govuk-grid-column-three-quarters
+      h1.dfe-heading-xl class='govuk-!-margin-bottom-4'
+        = t('gov_one_info.hero_header')
+      p.govuk-body-l = t('gov_one_info.hero_body')
+
 
 .govuk-grid-row
-  .govuk-grid-column-one-half
-    = m('gov_one_info.body')
+  .govuk-grid-column-three-quarters
```

Comment:
change this to column-full

Reply:
[3b7ba6a](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/968/commits/3b7ba6a7d3cbae5a3158c4a5ed1553e4f7086c90)